ptbr/tests/test_validation.py

```
@@ -0,0 +1,433 @@
+           f"{len(result.labels)} labels")
+    report("prepare sets source", result.source == str(sample))
+
+
+# ===================================================================
+# 6. CLI smoke test
+# ===================================================================
+
+def _run_validate_cli(path):
+    if importlib.util.find_spec("typer") is None:
+        return None
+    return subprocess.run(
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)